openwisp_radius/management/commands/base/delete_old_radiusbatch_users.py

```diff
@@ -16,15 +16,29 @@ def add_arguments(self, parser):
         parser.add_argument(
             '--older-than-days',
             action='store',
+            type=int,
+            help='Delete users that are older than days provided',
+        )
+        parser.add_argument(
+            '--older-than-months',
+            action='store',
+            type=int,
             default=BATCH_DELETE_EXPIRED,
```

Comment:
Can we update the default so that it's used only on the new argument? The default for this old argument should be `None`, so that the new argument will be used by default.

Reply:
Yes this could be done but the action would be same if no argument is provided that is why I did not changed it.I can do this in the next commit.